dataflow/src/main/java/org/checkerframework/dataflow/analysis/UnusedAbstractValue.java

```diff
@@ -0,0 +1,18 @@
+package org.checkerframework.dataflow.analysis;
+
+import org.checkerframework.javacutil.BugInCF;
+
+/**
+ * UnusedAbstractValue is an AbstractValue but is not involved in any lub computation during
+ * dataflow analysis. For those analyses which handle lub computation at a higher level (e.g., store
+ * level), it is sufficient to use UnusedAbstractValue and unnecessary to implement another specific
+ * AbstractValue. Example analysis using UnusedAbstractValue is LiveVariable analysis. This is a
+ * workaround for issue https://github.com/eisop/checker-framework/issues/200
+ */
+public final class UnusedAbstractValue implements AbstractValue<UnusedAbstractValue> {
+
```

Comment:
Can you make the default constructor for this class explicit and private and have it throw an exception? This class should never be instantiated, right?
For an example see https://github.com/eisop/checker-framework/blob/master/javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java#L101

Reply:
Added.